src/main/msp/msp_build_info.c

```diff
@@ -102,7 +102,7 @@ void sbufWriteBuildInfoFlags(sbuf_t *dst)
         BUILD_OPTION_AKK_SMARTAUDIO,
 #endif
 #ifdef USE_ALTITUDE_HOLD
-        BUILD_OPTION_ALT_HOLD_MODE,
+        BUILD_OPTION_ALTITUDE_HOLD,
```

Comment:
still need to be this irregardless, correct?

Reply:
@nerdCopter we are not supposed to this manually so we need to update cloud api instead. After that the script can be executed again.

@blckmn please update `USE_ALT_HOLDE_MODE` to `USE_ALTITUDE_MODE` in https://build.betaflight.com/api/options/4.6.0